Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -259,13 +264,15 @@ def calculate_indicator_score(self, risk_from_feed):
         Returns:
             int. The indicator's Dbot score
         """
-        dbot_score = 0
         risk_from_feed = int(risk_from_feed)
-        if risk_from_feed >= self.threshold or risk_from_feed >= 65:
+        if risk_from_feed >= self.malicious_threshold:
             dbot_score = 3
-        elif risk_from_feed >= 5:
+        elif risk_from_feed >= self.suspicious_threshold:
```

Comment:
Let's make **suspicious_threshold** default value 5 so we won't break bc for customers.

Reply:
I understand not wanting to break backward compatibility for other customers, but the suspicious threshold starts at 25 according to Recorded Future's documentation
![image](https://github.com/demisto/content/assets/32545292/72edda5a-a8fd-40f2-abbb-28c4f24b51dc)
https://support.recordedfuture.com/hc/en-us/articles/115000897208